docs/UserGuide.md

```diff
@@ -211,98 +234,47 @@ Therefore, to avoid unintentionally losing any information while editing tags, w
+    
+| Error message | Why it happens | Fix |
+|---------------|----------------|-----|
+| `Invalid command format!` | The command you input does not follow the specified format | Ensure the command you entered follows the following format: `find NAME...` where `NAME` is the specified name of the employee to search (case-insensitive).
+                                                                                                                                                                                            
+* **For advanced users:**
+  * Employees are displayed in the order in which they are arranged in the employee list.
 
-* **What It Does:**
-  * Finds all employees whose name contains at least one word in the search query (case-insensitive). Only name
-    containing the whole word are matched.
-* **Format:**
-  * `find SEARCH_QUERY...`
-* **Examples:**
+#### Finding employees by all specified tags: `find-all-tag`
 
-| S/N | Command information                                                                                                                                                                                                                  |
-|-----|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 1   | **Command:** `find martinez` <br><br> **Output:** Returns a list of employees whose names contain "*martinez*" e.g. "*Emiliano <u>Martinez</u>*" and "*Lisandro <u>Martinez</u>*"                                                    |
-| 2   | **Command:** `find harry redknapp` <br><br> **Output:** Returns a list of employees whose names contain either "*harry*" or "*redknapp*" e.g. "*<u>Harry</u> Kane*", "*Jamie <u>Redknapp</u>*", and "*<u>Harry</u> <u>Redknapp</u>*" |
-| 3   | **Command:** `find martin` <br><br> **Output:** Returns a list of employees whose names contain "*martin*" e.g. "*<u>Martin</u> Odegaard*" but not "*Gabriel Martinelli*" (only full word matches are displayed)                     |
-* **Acceptable Values:**
-  * Search query: A non-empty input is required. Inputs can contain one or more words.
-* **Expected Output on Success:**
-  * GUI Changes: A list of employees whose names contain at least one of the words
-  * Message shown to user: "[Number of matches] employees matched:"
-* **Expected Output on Failure:**
-  * `Invalid Command Format` - if an empty input is provided
+1. Type in the following [command](#glossary) in the [command box](#glossary) `find-all-tag t/TAG [t/MORE_TAGS]...` where `TAG` is the name of the [tags](#glossary) (case-sensitive), `t/TAG` representing at least one tag should be entered to search and `[t/MORE_TAGS]...` representing optional more tags to be entered. Note that only tags containing the whole word are matched.
```

Comment:
I think this can be rephrased to convey that the user has to key in at least one tag, but if the user would like to match more than one tag, he/she can add more tags at the back.